pref-adjust.lua

```diff
@@ -91,16 +136,201 @@ function clearpref_all_dwarves()
     end
 end
 -- ---------------------------------------------------------------------------
-function adjust_all_dwarves()
+function adjust_all_dwarves(profile)
     for _,v in ipairs(df.global.world.units.active) do
         if v.race == df.global.ui.race_id then
             print("Adjusting "..dfhack.TranslateName(dfhack.units.getVisibleName(v)))
-            brainwash_unit(v)
+            brainwash_unit(v,profile)
         end
     end
 end
+-- ---------------------------------------------------------------------------
+function build_all_lists(printflag)
+    list_of_inorganics={} -- Type 0 "Likes iron.."
```

Comment:
I commented [below](https://github.com/DFHack/scripts/pull/92/files#diff-d46c24224e6cb0ecfa2dab848b3c2190R186) before I saw these `list_of_*` tables in multiple places. Please remove them if you're not going to use them - they're not necessary in most cases, either, given how DF enum types can be indexed.

Reply:
I missed the fact that the `list` option uses these. This makes a lot of sense, actually!